--- nedresourcecacheorig.cc	2026-01-17 12:00:51.496036976 +0000
+++ nedresourcecache.cc	2026-01-17 12:01:46.325413048 +0000
@@ -291,9 +291,8 @@
 
 void NedResourceCache::doneLoadingNedFiles()
 {
-    if (doneLoadingNedFilesCalled)
-        throw NedException("NedResourceCache::doneLoadingNedFiles() may only be called once");
-    doneLoadingNedFilesCalled = true;
+    if (!doneLoadingNedFilesCalled)
+    	doneLoadingNedFilesCalled = true;
 
     // collect package.ned files
     for (auto& entry : nedFiles) {
